docs/_static/searchtools.js

```diff
@@ -257,8 +252,25 @@ var Search = {
       }
     });
 
+    function _getBreadcrumbs(item, linkUrl) {
+      let parentTitles = item[6];
+      let path = item[0].split('/')
+        .slice(0, -1);
+      path = path.map((el, index) => {
+        return {
+          "path": path.slice(0, index+1).join('/'),
+          "title": parentTitles[index]
+        }
+      })
+      let markup = path
+        .map((el, idx) => {
+            return idx===0 ? `<a href="/${el.path}"><b>${el.title}</b></a>` : `<a href="/${el.path}">${el.title}</a>` 
```

Comment:
Remove bold-ing of title.

```suggestion
            return idx===0 ? `<a href="/${el.path}">${el.title}</a>` : `<a href="/${el.path}">${el.title}</a>` 
```